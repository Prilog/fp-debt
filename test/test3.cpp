int main() {
  string cur = "";
  bool flag = true;
  while (flag) {
    getline(cin, cur);
    cout << "Have read " + cur + "\n";
    if (cur == "STOP") {
      flag = false;
    } else {
      flag = true;
    }
  }
  return 0;
}