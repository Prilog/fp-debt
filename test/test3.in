exe
a
b
c
d
e
STOP