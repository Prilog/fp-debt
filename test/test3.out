=================
print required action:
"exe" - execute file CDSLResult.hs
*filename* - interpretes CDSLResult.hs into file *filename*
Have read a
Have read b
Have read c
Have read d
Have read e
Have read STOP
main returned with exit code: 0
